src/dao/PourChartDAO.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import type { PourChartParams, PourChartDataItem } from '../index';
+import type DAOResult from './DAOResult';
+
+import DAO from './DAO';
+import { DAO_ENTITIES } from '../constants';
+import DefaultTranslator from '../translators/DefaultTranslator';
+
+class PourChartDAO extends DAO<PourChartDataItem, PourChartDataItem> {
```

Comment:
again, not sure about the name of ``PourChartDateItem` but call that just PourChart seems weird.